### ISSUE TITLE
className on a Space percolating to inner div too

### ISSUE THREAD

**Issue by thekapadia** [aeagle/react-spaces#30]
@aeagle I have the following.

`<Left className={"my-class"} />`

The resulting DOM is formed as follows.

```
<div class="spaces-space anchor-left my-class">
  <div class="spaces-space-inner my-class"></div>
</div>
```

Ideally, `my-class` should be applied to only the root element, not the inner element. This causes a problem when let's say, `"my-class": { border: 1px solid red }` -- this gives me a double border.

**Comment by thekapadia** [comment]
@aeagle This issue is reproducible in this scenario.
https://codesandbox.io/s/bitter-brook-w7g24?fontsize=14

I get the double border, on the _container_ as well as _inner_.

**Comment by aeagle** [comment]
@thekapadia I've published a fix for this in version 0.1.17. Let me know that this fixes your issue.
